libs/cri-response-service/src/main/java/uk/gov/di/ipv/core/library/service/CriResponseService.java

```diff
@@ -62,15 +67,37 @@ public void persistCriResponse(
         dataStore.create(criResponseItem, ConfigurationVariable.CRI_RESPONSE_TTL);
     }
 
-    public CriResponseItem getFaceToFaceRequest(String userId) {
-        return dataStore.getItem(userId, "f2f");
-    }
-
     public void deleteCriResponseItem(String userId, Cri credentialIssuer) {
         dataStore.delete(userId, credentialIssuer.getId());
     }
 
     public void updateCriResponseItem(CriResponseItem responseItem) {
         dataStore.update(responseItem);
     }
+
+    @SuppressWarnings("java:S4276")
```

Comment:
This is for passing `getVcIat` as a less-precise `Function`, but it allows a `null` result which is valid as F2F VC doesn't need `iat`.